hydra-node/src/Hydra/Chain/Direct/Tx.hs

```diff
@@ -314,6 +314,60 @@ collectComTx networkId scriptRegistry vk headId headParameters (headInput, initi
   commitRedeemer =
     toScriptData $ Commit.redeemer Commit.ViaCollectCom
 
+-- | Construct a _decrement_ transaction which takes as input some 'UTxO' present
+-- in the L2 ledger state and makes it available on L1.
+decrementTx ::
+  -- | Published Hydra scripts to reference.
+  ScriptRegistry ->
+  -- | Party who's authorizing this transaction
+  VerificationKey PaymentKey ->
+  -- | Head identifier
+  HeadId ->
+  -- | Parameters of the head.
+  HeadParameters ->
+  -- | Everything needed to spend the Head state-machine output.
+  (TxIn, TxOut CtxUTxO) ->
+  -- | Decrement UTxO.
+  UTxO ->
+  -- | Confirmed Snapshot
+  Snapshot Tx ->
```

Comment:
Should: remove duplication between `UTxO` and `Snapshot`

the `utxoToDecommit` are already included in the `Snapshot`